packages/account/src/Components/poi/status/unsupported/unsupported.jsx

```diff
@@ -28,6 +31,22 @@ const Unsupported = ({ country_code, handlePOIforMT5Complete, ...props }) => {
         country_code,
     });
 
+    if (props?.manual) {
```

Comment:
Can you de-structure `manual`, `redirect_button`, `needs_poa` and `handleRequireSubmission` from the props? It makes the code more readable this way as the reader knows what props are used and what props are simply forwarded